os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -920,7 +921,7 @@ static void _up_assert(int errorcode)
 {
 	/* Are we in an interrupt handler or the idle task? */
 
-	if (g_upassert || current_regs || (this_task())->pid == 0) {
+	if (current_regs || (this_task())->pid == 0) {
```

Comment:
Do you intend removing `g_upassert`? If you remove it, it is not used anymore. Let's remove definition and other location as well.

Reply:
But as far as I know, it makes avoiding recursive call. Is it ok without it?

Reply:
yes, I will modify the code to make it macro protected, so this changes will come under fault manager macro and not in general.